aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/transfer/TransferWorkerObserver.kt

```diff
@@ -186,8 +186,14 @@ internal class TransferWorkerObserver private constructor(
                 )
             )?.use {
                 while (it.moveToNext()) {
```

Comment:
if we catch and continue, what happens on the next `it.moveToNext()` ? does it return null and break out? would it be safer to catch and break out?

Reply:
It would be false the next time and break out itself

Reply:
It is not reported to throw: https://developer.android.com/reference/android/database/Cursor#moveToNext()

Reply:
I went ahead and added a break as looking through cursor code, I couldn't find where cursor count would be reset to 0 when closed, and didn't see moveToNext checking if cursor were open.